src/vcpkg/commands.applocal.cpp

```diff
@@ -0,0 +1,550 @@
+                deployPluginsQt("iconengines", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("platforminputcontexts", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("styles", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5Networkd.dll" || target_binary_name == "Qt5Network.dll")
+            {
+                deployPluginsQt("bearer", target_binary_dir, qt_plugins_dir);
+
+                std::error_code ec;
+                std::vector<Path> children = m_fs.get_files_non_recursive(bin_dir, ec);
+                for (auto&& c : children)
+                {
+                    const std::string& c_filename = c.filename().to_string();
+                    if (Strings::starts_with(c_filename, "libcrypto-") && Strings::ends_with(c_filename, ".dll"))
+                    {
+                        deploy_binary(target_binary_dir, bin_dir, c_filename);
+                    }
+
+                    if (Strings::starts_with(c_filename, "libssl-") && Strings::ends_with(c_filename, ".dll"))
+                    {
+                        deploy_binary(target_binary_dir, bin_dir, c_filename);
+                    }
+                }
+            }
+            else if (target_binary_name == "Qt5Sqld.dll" || target_binary_name == "Qt5Sql.dll")
+            {
+                deployPluginsQt("sqldrivers", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5Multimediad.dll" || target_binary_name == "Qt5Multimedia.dll")
+            {
+                deployPluginsQt("audio", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("mediaservice", target_binary_dir, qt_plugins_dir);
+                deployPluginsQt("playlistformats", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5PrintSupportd.dll" || target_binary_name == "Qt5PrintSupport.dll")
+            {
+                deployPluginsQt("printsupport", target_binary_dir, qt_plugins_dir);
+            }
+            else if (target_binary_name == "Qt5Qmld.dll" || target_binary_name == "Qt5Qml.dll")
+            {
+                std::error_code ec;
+                if (!m_fs.exists(target_binary_dir / "qml", ec))
+                {
+                    if (m_fs.exists(bin_dir / "../qml", ec))
+                    {
+                        m_fs.copy_regular_recursive(bin_dir / "../qml", target_binary_dir, VCPKG_LINE_INFO);
+                    }
+                    else if (m_fs.exists(bin_dir / "../../qml", ec))
+                    {
+                        m_fs.copy_regular_recursive(bin_dir / "../../qml", target_binary_dir, VCPKG_LINE_INFO);
+                    }
+                    else
+                    {
+                        Checks::exit_with_message(VCPKG_LINE_INFO, "FAILED", ec.message());
+                    }
+                }
```

Comment:
Is this missing a foreach? https://github.com/microsoft/vcpkg/blob/8d9c52191e039fe49c580807efee502d22f32786/ports/qt5-base/qtdeploy.ps1#L76